dist/vendor.sh

```diff
@@ -26,8 +26,10 @@ cargo vendor > "${CARGO_CONFIG}"
 # some vendored files aren't world readable which has lead to issues in container builds when trying out the workflow
 find ./vendor -type f -exec chmod a+r {} \;
 
-git add --force "${FILES_TO_COMMIT[@]}"
```

Comment:
Prow would run the job with [CI=true env var](https://github.com/kubernetes/test-infra/blob/master/prow/jobs.md#job-environment-variables), so if its set we should exit with 0